tutorials/editor/script_editor.rst

```diff
@@ -0,0 +1,411 @@
+  the file and jumps to the respective line.
+- **Replace in Files...**: Opens a window to search and replace text with different text within the 
+  found files in the project folder. After clicking "Replace...", you can select in which files to 
+  replace using the "Search Results" tab in the bottom panel by (un)checking them and using 
+  "Replace All" button.
+  
+.. image:: img/editor_ui_script_editor_replaceinfiles.webp
+
+.. warning:: Note that "Replace in Files" operation cannot be undone!
+
+.. tip:: Both the `Find in Files` and `Replace in Files` windows share the "Search..."
+  and "Replace..." buttons. The only difference in the latter window is an additional text
+  field that automatically fills in the search results panel when the "Replace..." button
+  is clicked. The replacement operation is only executed if you click the "Replace All"
+  button in this bottom panel, allowing you to also edit the word to replace later within
+  this panel.
+
+.. image:: img/editor_ui_script_editor_replace_all.webp
+
+- **Contextual Help***: Opens the list of built-in class reference similar to pressing F1
+  on a symbol, or choosing "Lookup Symbol" from the context menu.
+
+The **Go To** menu lets you navigate within the code at ease with these options:
+
+.. image:: img/script_editor_icons/text_editor_goto_menu.webp
+
+- **Go to Function...**: Opens the function list to jump to. You can achieve the same result
+  by typing in the filter methods bar in the script panel.
+- **Go to Line...**: Jumps to the entered line number in the code editor.
+- **Bookmarks**: Contains actions for the bookmark functionality, which you can use to find
+  your way through your code easier, such as an incomplete section. Bookmarked lines will
+  have a blue bookmark symbol left of the line number.
+
+  - `Toggle Bookmark*`: Adds or removes the bookmark on the line where the caret is. You can
+    also right click on a line to achieve this.
+  - `Remove All Bookmarks`: Removes all bookmarks in the open document.
+  - `Go to Next Bookmark`: Jumps to the next bookmark in the open document.
+  - `Go to Previous Bookmark`: Jumps to the previous bookmark in the open document.
+  - `Bookmarks` menu will also contain the list of bookmarked lines, including their line
+    number and displaying the partial content in that line.
+
+- **Breakpoints**: Breakpoints are helpful while debugging your code. Similar to "Bookmarks"
+  menu, this menu lets you add or remove breakpoints, navigate between them and directly
+  jump to a specific breakpoint. An easy way to add a breakpoint is hovering over the blank area
+  left of a line number. It will show a faded red circle. Clicking it will add a
+  breakpoint and the circle will stay there. Clicking on a circle removes the breakpoint.
+
+**Debug** menu offers actions which can be used while debugging. See
+:ref:`doc_debugger_tools_and_options` for more.
+
+.. _doc_script_editor_coding_area:
+
+Coding area
+-----------
```

Comment:
Is this name used anywhere else, prior to this page existing?
And is there some other name used for it in the editor itself?

We have to be a little careful with giving things names.

Reply:
Unfortunately no. What do you suggest using for this section?

Reply:
Let's:
- Double check if it has a specific name anywhere else in the docs, or in the engine itself
- Maybe ask in the dev chat if there is another existing name for it.
- *Consider* avoiding specifically referring to this part of the UI. There is a chance that the proper name for this might just be "script editor" or "text editor". since it is the *main* part of the Script Editor workspace. There might not actually be a name for it like there is for the viewport for 2D and 3D.
- Consider calling it the "coding area", but avoid calling the section header "Coding area". If we get the name wrong for a while in the official docs it's *fine*, but titles of sections are slightly harder to change.

Reply:
Thanks. I will leave it as it is now.

Reply:
I would not hold up this PR for this issue, but I've never seen this called anything but "script editor".

Reply:
I've decided this isn't worth changing (at this moment). Let's go ahead and merge.